packages/block-editor/src/components/keyboard-shortcuts/index.js

```diff
@@ -142,20 +135,6 @@ function KeyboardShortcuts() {
 		{ isDisabled: clientIds.length < 2 }
 	);
 
-	useShortcut(
-		'core/block-editor/select-all',
-		useCallback(
-			( event ) => {
-				event.preventDefault();
-				multiSelect(
```

Comment:
I only saw now that these shortcuts are duplicated here, while also being in writing flow. This doesn't make any sense. The select all shortcut should only work from a focussed block upwards. These global shortcuts will also give problems when you focus something outside the block editor and press cmd+a. In other words, it should never be a global behaviour. Additionally it will give problems in the iframe editor when we remove event bubbling through the iframe.

Reply:
There are some related issues. I found https://github.com/WordPress/gutenberg/issues/19102 and  https://github.com/WordPress/gutenberg/issues/22689 in a quick search.

I think `registerShortcut` should ideally still be called as I believe that makes the shortcut display in the shortcut help modal. And then the code in `useSelectAll` could reference that registered shortcut.

That would also have the side effect of allowing a plugin to unregister the shortcut or change the shortcut keys. This is an interesting one because it uses the same key combination as the browser shortcut, so there's a question over how that would work..

Reply:
You're right, I modified it to use registerShortcut again.